SystemInfo/SystemInfoCollectorRegistry.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * File containing the SystemInfoCollectorRegistry interface.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+/**
+ * A registry of SystemInfoCollectors.
+ */
+interface SystemInfoCollectorRegistry
+{
+    /**
+     * Adds SystemInfoCollectors, each with an identifier string, to the registry.
+     *
+     * @param \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector[] $items A hash of SystemInfoCollectors, with identifier string as key.
+     */
+    public function addItems(array $items);
```

Comment:
What bothers me a bit with using the ctor is that this aspect of the service is _not_ part of the interface... 

But it is also the simplest approach, so why not.